code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -38,7 +38,8 @@
 	var/subscreen			// Which specific function of the main screen is being displayed
 
 	var/obj/item/pda/ai/pai/pda = null
-
+	var/obj/machinery/newscaster		// A newscaster

```

Comment:
having a machine inside a mob. What could possibly go wrong.

Reply:
The eventual plan here is to have an entire space station within a pAI. I call it Project Turtles: All the Way Down

Reply:
Are tiny fedoras included?

Reply:
*nervous laughing*